class Error(Exception):
    def __init__(self, message):
        pass
